eng/innerloop.yml

```diff
@@ -5,7 +5,7 @@ parameters:
   queue: {}
```

Comment:
I don't think you're missing anything. I'm fairly certain these files are remnants of work that was done in this branch before i got it. I didn't delete them because I thought they may still be useful once I handed this off